src/modules/commander/commander_params.c

```diff
@@ -228,10 +228,11 @@ PARAM_DEFINE_INT32(COM_HOME_IN_AIR, 0);
 /**
  * RC control input mode
  *
- * The default value of 0 requires a valid RC transmitter setup.
- * Setting this to 1 allows joystick control and disables RC input handling and the associated checks. A value of
- * 2 will generate RC control data from manual input received via MAVLink instead
- * of directly forwarding the manual input data.
+ * With value 0 a valid RC transmitter calibration is required.
+ * A value of 1 allows joystick control only and disables RC input handling and the associated checks.
+ * A value of 2 allows both sources and will fall back to the next valid one once the stream in use gets invalid.
```

Comment:
Is it that the stream is _invalid_, or "lost"? I.e. If go out of range of RC but I have a good telemetry radio will it switch to using  joystick? "Invalid" kind of implies that the signal is incorrect, not so much that it is lost.

Also, what RC checks are required in this case? Do I still need to do an RC calibration.

Suggestion below, but might need to be modified in light of responses to ^^^

```suggestion
 * A value of 2 allows either RC Transmitter or Joystick input. The first valid input is used, will fallback to other sources if the input stream becomes invalid.
```

Reply:
> Is it that the stream is invalid, or "lost"?

By invalid I meant no sample was received for `COM_RC_LOSS_T` seconds. Thanks for clarifying.

> Also, what RC checks are required in this case? Do I still need to do an RC calibration.

RC calibration is currently only required with `COM_RC_IN_MODE` 0. We should change that in the future to when RC is actually enabled in the groundstation and have the `COM_RC_IN_MODE` option set accordingly. Currently we need to have options that allow the user to use both because where should he know from that there's the other one for each case and how to configure it. It needs to get intuitive to configure from the QGC UI.